Makefile

```diff
@@ -172,6 +172,11 @@ ready: manifests generate fmt fmt_license go.mod vet lint test ## Make the code
 test: manifests generate envtest ## Run unit tests
 	$(K8S_CLI) apply -k ./config/crd
 	GOMEGA_DEFAULT_EVENTUALLY_TIMEOUT=10s KUBEBUILDER_ASSETS="$(shell $(ENVTEST) --arch=amd64 use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out -covermode=atomic -coverpkg=./...
+	make -C pact -f Makefile test-pact
+
+unit-tests: manifests generate envtest ## Run unit tests
```

Comment:
One of the main benefits of pact tests is that they run together with unit tests. By design, they should be part of the codebase and run natively by executing `go test` together with unit tests. We're hacking this and excluding them to separate module, but I understand you don't want to include new dependencies. Excluding them even from the `make test` command seems like a step back - it would make devs run two separate commands for testing and make them spend more time here. If, for any reason, devs want to run just unit tests, they have an option by using this `unit-tests` target.

Reply:
Ok, I've created an issue for tracking [HAC-5509](https://issues.redhat.com/browse/HAC-5509). I'll update the code.